code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -56,12 +56,15 @@
 	///Whether it can be painted
 	var/paintable = FALSE
 
-	///Is the thing being rebuilt by SSair or not. Prevents list blaot
+	///Is the thing being rebuilt by SSair or not. Prevents list bloat
 	var/rebuilding = FALSE
 
+	//Default is to allow ventcrawling and seeing pipes
+	var/vent_movement = VENTCRAWL_ALLOWED | VENTCRAWL_CAN_SEE
```

Comment:
Doc-comment this.

Reply:
Did all the other suggested changes but I'm not sure what you meant by documenting this. Let me know if the comment on top is not enough / you want me to add to it.

Reply:
Doc comments are `///`, but this doesn't describe what this is, just what the defaults are.